.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 		es6: true
 	},
 	parserOptions: {
-		ecmaVersion: 2018
+		ecmaVersion: 2021
```

Comment:
Don't touch this

Reply:
Without this parser can't handle `import.meta.url` [here](https://github.com/webpack/webpack/pull/16305/files#diff-7b9c6a5220df8c4fc9ecb1eeecf91297e19e97b33651306e2c5118b4851d8553R8). Do you know how to make it work without upgrading ECMA parser's version? See this example of error: https://github.com/eslint/eslint/issues/11189

Reply:
Just override it for file, no need to change it globally, because it is not safe to change for all code (new contributes will use a new syntax and it will break code for old Node.js versions)